Add collection section

Sinfinka opened the pull request:
HTML, CSS for mobile, tablet, desktop and desktop-max. Add imgs.

Sinfinka committed:
Add collektion

HTML, CSS for mobile, tablet, desktop and desktop-max. Add imgs.

AiHikki commented on src/css/collection.css:
```diff
@@ -0,0 +1,108 @@
+
+.collection-card-caption {
+  padding-top: 18px;
+  margin-bottom: 20px;
```
Margin-bottom пробиває батьківський елемент, і margin-bottom потрібен тільки на мобільній версії. Можна зробити відступ між елементами за допомогою
.collection-card-item:not(:last-child) {margin-bottom: 20px;}

Sinfinka replied:
Внесла зміни.

Sinfinka committed:
Fixed

Add 2x background imgs, fixed margin-bottom in cards. Add font-size to pharagraph.
